internal/datastore/postgres/postgres.go

```diff
@@ -477,25 +477,29 @@ func currentlyLivingObjects(original sq.SelectBuilder) sq.SelectBuilder {
 	return original.Where(sq.Eq{colDeletedXid: liveDeletedTxnID})
 }
 
-// defaultCustomPlan parses a Postgres URI and determines if a plan_cache_mode
-// has been specified. If not, it defaults to "force_custom_plan".
+// defaultQueryExecMode parses a Postgres URI and determines if a default_query_exec_mode
+// has been specified. If not, it defaults to "cache_describe".
 // This works around a bug impacting performance documented here:
 // https://spicedb.dev/d/force-custom-plan.
-func defaultCustomPlan(poolConfig *pgxpool.Config) (*pgxpool.Config, error) {
-	if existing, ok := poolConfig.ConnConfig.Config.RuntimeParams["plan_cache_mode"]; ok {
+//
+// The docs for the different execution modes offered by pgx may be found
+// here:
+// https://pkg.go.dev/github.com/jackc/pgx/v5#QueryExecMode
+func defaultQueryExecMode(poolConfig *pgxpool.Config) *pgxpool.Config {
+	if existing, ok := poolConfig.ConnConfig.Config.RuntimeParams["default_query_exec_mode"]; ok {
```

Comment:
This won't work because pgx deletes this param from `RuntimeParams`
https://github.com/jackc/pgx/blob/cf6ef75f916648857cf7b46322d6d7af7d372917/conn.go#L173

I need to find another way to determine if the user omitted this vs wants to override